build/DependencyVersions.props

```diff
@@ -26,22 +26,22 @@
     <MicrosoftCodeAnalysisBuildTasksPackageVersion>$(MicrosoftCodeAnalysisCSharpPackageVersion)</MicrosoftCodeAnalysisBuildTasksPackageVersion>
     <MicrosoftNetCompilersNetcorePackageVersion>$(MicrosoftCodeAnalysisCSharpPackageVersion)</MicrosoftNetCompilersNetcorePackageVersion>
-    <MicrosoftNETSdkPackageVersion>2.2.100-preview1-63215-01</MicrosoftNETSdkPackageVersion>
+    <MicrosoftNETSdkPackageVersion>2.2.100-preview2-63312-02</MicrosoftNETSdkPackageVersion>
     <MicrosoftNETBuildExtensionsPackageVersion>$(MicrosoftNETSdkPackageVersion)</MicrosoftNETBuildExtensionsPackageVersion>
     <MicrosoftNETSdkRazorPackageVersion>2.1.1</MicrosoftNETSdkRazorPackageVersion>
-    <MicrosoftNETSdkWebPackageVersion>2.1.400-preview1-20180705-1834985</MicrosoftNETSdkWebPackageVersion>
+    <MicrosoftNETSdkWebPackageVersion>2.2.100-preview2-20180911-2023461</MicrosoftNETSdkWebPackageVersion>
     <MicrosoftNETSdkPublishPackageVersion>$(MicrosoftNETSdkWebPackageVersion)</MicrosoftNETSdkPublishPackageVersion>
     <MicrosoftNETSdkWebProjectSystemPackageVersion>$(MicrosoftNETSdkWebPackageVersion)</MicrosoftNETSdkWebProjectSystemPackageVersion>
-    <MicrosoftDotNetCommonItemTemplatesPackageVersion>1.0.2-beta4-20180803-1918431</MicrosoftDotNetCommonItemTemplatesPackageVersion>
+    <MicrosoftDotNetCommonItemTemplatesPackageVersion>1.0.2-beta4-20180911-2023462</MicrosoftDotNetCommonItemTemplatesPackageVersion>
     <MicrosoftDotNetCommonProjectTemplates22PackageVersion>$(MicrosoftDotNetCommonItemTemplatesPackageVersion)</MicrosoftDotNetCommonProjectTemplates22PackageVersion>
     <MicrosoftDotNetTestProjectTemplates22PackageVersion>1.0.2-beta4-20180821-1966911</MicrosoftDotNetTestProjectTemplates22PackageVersion>
-    <MicrosoftTemplateEngineCliPackageVersion>1.0.2-beta3</MicrosoftTemplateEngineCliPackageVersion>
+    <MicrosoftTemplateEngineCliPackageVersion>1.0.2-beta4-20180911-2023462</MicrosoftTemplateEngineCliPackageVersion>
     <MicrosoftTemplateEngineAbstractionsPackageVersion>$(MicrosoftTemplateEngineCliPackageVersion)</MicrosoftTemplateEngineAbstractionsPackageVersion>
     <MicrosoftTemplateEngineCliLocalizationPackageVersion>$(MicrosoftTemplateEngineCliPackageVersion)</MicrosoftTemplateEngineCliLocalizationPackageVersion>
     <MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion>$(MicrosoftTemplateEngineCliPackageVersion)</MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion>
     <MicrosoftTemplateEngineUtilsPackageVersion>$(MicrosoftTemplateEngineCliPackageVersion)</MicrosoftTemplateEngineUtilsPackageVersion>
-    <MicrosoftDotNetPlatformAbstractionsPackageVersion>2.2.0-preview1-26609-02</MicrosoftDotNetPlatformAbstractionsPackageVersion>
-    <MicrosoftExtensionsDependencyModelPackageVersion>2.2.0-preview1-26609-02</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftDotNetPlatformAbstractionsPackageVersion>2.1.0</MicrosoftDotNetPlatformAbstractionsPackageVersion>
```

Comment:
@livarcocc Probably not, checking.

Reply:
@weshaggard @wtgodbe It looks like Microsoft.DotNet.PlatformAbstractions was produced for 2.2 briefly until around june 20 and probably stopped being produced because of this commit:

https://github.com/dotnet/corefx/commit/8dabbd8adc40cec5b12b2d2168fd4ef3e9f08a62

Should we have a new version of these packages in 2.2?

Reply:
That package ships from core-setup and it hasn't had any changes so it shouldn't need to be updated for 2.2.

Reply:
@mmitche don't we build PlatformAbstractions out of core-setup? I'm confused how that commit would stop us from building PlatformAbstractions. Looking at the 2-1 orchestrated builds in dotnetfeed, this package last appeared on May 28, and was first absent on May 30. I'll go look at the change history for that time period.

Reply:
https://github.com/dotnet/core-setup/pull/4172 (and its 2.2 port in https://github.com/dotnet/core-setup/pull/4377) is the culprit. I moved PlatformAbstractions & DependencyModel out of the list of packages to build for servicing with that change. Looks like PlatformAbstractions hasn't had any changes since then, and the only change to DependencyModel was in the spelling of a variable name (https://github.com/dotnet/core-setup/commit/84c9d2d68103c8bac4a432e2cfc2073ba24f1924#diff-ed7d78f6b4613a62e8d2ef98d9ca459b), so my inclination is that we are doing the right thing by not building these for servicing releases. @weshaggard do you agree?

Reply:
>@weshaggard do you agree?

Yes we should stick to 2.1 version.